### DISCUSSION TITLE
Comments in Markdown support

### DISCUSSION THREAD

**Discussion by olayway** [flowershow/flowershow#394] [General and Help]
Would be nice to support comments that do not render.

**e.g. Obsidian support this:**

![image](https://user-images.githubusercontent.com/52197250/179369113-ba661696-72c1-4421-acae-b2db2b28e13b.png)

**Flowershow currently does not but supports MDX comments (?):**

![image](https://user-images.githubusercontent.com/52197250/179369099-feb15584-cf63-4f02-ab3b-f3dda252fc53.png)

Given we have MDX you could just use html/jsx commenting which mdx supports i believe see e.g. https://github.com/mdx-js/mdx/issues/184

My suggestion here is we just add a page to document this e.g. `docs/comments`

## Acceptance

* [ ] docs page on comments e.g. `docs/comments` with example/test

## Tasks

* [ ] Test that MDX supported comments are indeed supported e.g. html or jsx style
* [ ] Put up a docs page

**Comment by rufuspollock** [comment]
Given we have MDX you could just use html/jsx commenting which mdx supports i believe see e.g. https://github.com/mdx-js/mdx/issues/184

My suggestion here is we just add a page to document this e.g. `docs/comments`

**Comment by sahilm-10** [comment]
@rufuspollock @olayway Can you guide me ? I want to contribute. I am new to Opensource

**Comment by rufuspollock** [comment]
@sahilm-10 what do you understand of the issue for yourself?

**Comment by sahilm-10** [comment]
Some docs page needs to be added along with MDX supported comments @rufuspollock . Am i right?

**Comment by rufuspollock** [comment]
@sahilm-10 yes i think the start is to research docs support in mdx and then add a page docs/comments to flowershow.app exclaiming how to add comments to pages ...

**Comment by olayway** [comment]
Documented here https://flowershow.app/docs/syntax#comments
